content/en/docs/pipelines/installation/overview.md

```diff
@@ -11,6 +11,7 @@ with each option:
 * A [standalone](#standalone) Kubeflow Pipelines deployment.
 * Kubeflow Pipelines as [part of a full Kubeflow deployment](#full-kubeflow).
 * **Alpha**: [GCP Hosted ML Pipelines](#marketplace).
+* A [local](#localcluster) Kubeflow Pipelines deployment for testing purposes.
```

Comment:
other `#full-kubeflow` links are sections in this page, you don't need one here because you didn't add a section.

suggest adding a link to the local deployment guide:
```
[local](/docs/pipelines/installation/localcluster-deployment)
```